app/services/discord/community_message_service.rb

```diff
@@ -0,0 +1,52 @@
+module Discord
+  class CommunityMessageService
+    include RoutesResolvable
+    def initialize(community)
+      @community = community
+    end
+
+    def post_topic_created(topic)
+      return if topic.blank?
+
+      message =
+        I18n.t(
+          'services.discord.community_message_service.post_topic_created.message',
```

Comment:
Why not keep the message under the `post_topic_created` key? The `message` key doesn't have any siblings anyway. 🤔